hypervisor/common/schedule.c

```diff
@@ -13,6 +13,14 @@
 #include <schedule.h>
 #include <sprintf.h>
 
+/**
+ * @pre obj != NULL
+ */
+uint16_t sched_get_pcpuid(const struct thread_object *obj)
+{
```

Comment:
get running or previously running PCPU ID of the thread?

Reply:
get the id of PCPU which to run 'obj' .